src/pages/settings/Payments/TransferBalancePage.js

```diff
@@ -169,6 +163,43 @@ class TransferBalancePage extends React.Component {
     }
 
     render() {
+        if (this.props.walletTransfer.shouldShowSuccess && !this.props.walletTransfer.loading) {
+            return (
+                <ScreenWrapper>
+                    <KeyboardAvoidingView style={[styles.flex1]} behavior="height">
+                        <HeaderWithCloseButton
+                            title={this.props.translate('common.transferBalance')}
+                            onCloseButtonPress={PaymentMethods.dismissSuccessfulTransferBalancePage}
+                        />
+                        <View style={[styles.pageWrapper, styles.flex1, styles.flexColumn, styles.alignItemsCenter, styles.justifyContentCenter]}>
+                            <Icon
+                                src={Illustrations.TadaBlue}
+                                height={100}
+                                width={100}
+                                fill={defaultTheme.iconSuccessFill}
+                            />
+                            <View style={[styles.ph5]}>
+                                <Text style={[styles.mt5, styles.h1, styles.textAlignCenter]}>
+                                    {this.props.translate('transferAmountPage.transferSuccess')}
+                                </Text>
+                                <Text style={[styles.mt3, styles.textAlignCenter]}>
+                                    {this.props.translate('transferAmountPage.transferDetail')}
```

Comment:
We don't want to always show this right? Transferring to a debit card is instant

Reply:
That's a good point, re-opening the mockup issue to see if we can get another copy in the case of debit card transfers: https://github.com/Expensify/Expensify/issues/217387

Reply:
Just waiting on the Spanish translation and then we should be good to go!